keps/sig-auth/2799-reduction-of-secret-based-service-account-token/kep.yaml

```diff
@@ -0,0 +1,32 @@
+title: Reduction of Secret-based Service Account Tokens
+kep-number: 2799
+authors:
+  - "@zshihang"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+status: implementable
+creation-date: 2021-06-25
+last-updated: 2021-09-10
+reviewers:
+  - "@liggitt"
+  - "@mikedanese"
+approvers:
+  - "@liggitt"
+  - "@mikedanese"
+prr-approvers:
+  - TBD
+stage: alpha
+latest-milestone: "v1.23"
+milestone:
+  alpha: "v1.23"
+feature-gates:
+  - name: LegacyServiceAccountTokenNoAutoGeneration
+    components:
+      - kube-controller-manager
+  - name: LegacyServiceAccountTokenCleanUp
+    components:
+      - kube-controller-manager
```

Comment:
I suspect kube-apiserver will pay attention to this gate as well when writing last-used dates to secrets